clap_man/src/lib.rs

```diff
@@ -0,0 +1,105 @@
+#![doc(html_logo_url = "https://raw.githubusercontent.com/clap-rs/clap/master/assets/clap.png")]
+#![doc = include_str!("../README.md")]
+#![warn(missing_docs, trivial_casts, unused_allocation, trivial_numeric_casts)]
+#![forbid(unsafe_code)]
+
+mod man;
+mod render;
+
+use std::io::Write;
+
+/// Man page generator
+#[derive(Debug, Clone)]
+pub struct Man {
+    section: Option<i8>,
+    manual: Option<String>,
+    sections: Vec<(String, Vec<String>)>,
+}
+
+impl Default for Man {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+/// Generate manpage for your application using sane defaults.
+pub fn generate_manpage<'a>(app: &mut clap::App<'a>, buf: &mut dyn Write) {
+    let man = Man::new().section(1).manual("General Commands Manual");
```

Comment:
Should we explicitly set these or stick to defaults?

Reply:
This now uses `Man::default()`.